src/components/app.js

```diff
@@ -39,24 +34,32 @@ class App extends React.Component {
               <h3 className='qpp-docs-title-text'>QPP Submissions API Documentation</h3>
             </div>
           </div>
-          <div className='content-block'>
-            <div className='content-container ds-l-row'>
-              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
-                <ul className='ds-c-vertical-nav__subnav'>
+
+          <div className='content-container ds-l-row'>
```

Comment:
`content-container` is fine — but just pointing out there's also [`ds-l-container`](https://design.cms.gov/layout/grid/) which applies similar constraints (but it would need applied to a parent of `ds-l-row`)